corehq/apps/hqwebapp/urls.py

```diff
@@ -58,8 +58,6 @@
 domain_specific = [
     url(r'^$', redirect_to_default, name='domain_homepage'),
     url(r'^login/$', domain_login, name='domain_login'),
-    url(r'^login/mobile/$', domain_login, name='domain_mobile_login',
-        kwargs={'template_name': 'login_and_password/mobile_login.html'}),
```

Comment:
would it be possible to quickly pull analytics from nginx to just confirm that nothing hits this? it would probably be worth running on prod and prod india.

Reply:
```
jroth@hqproxy0:/home/cchq/www/production/log$ grep 'login/mobile' production_commcare-nginx_access.log* | wc -l
0
```

Reply:
nice, did you run on india as well?

Reply:
Oh, nope. I'll check that out now tho

Reply:
Setting up the vpn took a bit. Glad I have it now tho

```jroth@proxy0:/home/cchq/www/softlayer/log$ grep 'login/mobile' softlayer_commcare-nginx_access.log* | wc -l
0
```

Reply:
nice! this is g2g from my side. thanks!